README.md

```diff
@@ -159,6 +159,18 @@ The service on the tag, is so that if you have multiple REST endpoints you can d
 ### Symfony Debug Toolbar / Profiler
 <img src="/src/Resources/doc/img/debug_logs.png" alt="Debug Logs" title="Symfony Debug Toolbar - Guzzle Logs" style="width: 360px" />
 
+### Symfony logs
+All request are logged into symfony default logger (`@logger` service) with next format:
+```
+[datetime] eight_points_guzzle.LEVEL: {method} {uri} {code} 
```

Comment:
maybe LEVEL can be replaced with {log_level}?

Reply:
also [datetime]: [{datetime}]

Reply:
good idea :+1: